### DISCUSSION TITLE
Zone Template Variable Support

### DISCUSSION THREAD

**Discussion by bradley-baker** [ngoduykhanh/PowerDNS-Admin#1672] [Q&A] [answered]
I've got a zone template with our name servers in it which are static. I'd like to add an apex alias record that points the root of the domain to a www record but obviously, the record would change from domain to domain. 

So for example:
ourdomain.org 3600    IN      ALIAS   www.ourdomain.com

We want to create a record in our template similar to this:
Name: @ 
Type: ALIAS
Status: Active
TTL: 60
Data: www.$DOMAIN$

And when the zone template is used I want powerdns-admin to replace $DOMAIN$ with the actual domain. So say I was using the zone template to create a zone called test.com it would create a record:

Name: @ 
Type: ALIAS
Status: Active
TTL: 60
Data: www.test.com

This may seem like a trivial savings to not have to create one extra record but when creating dozens of zones every little bit helps.

Thanks
Brad

**Comment by lovelord83** [comment, marked as answer]
You have my 2 cents for this!

Any cPanel DNS has this feature turned on. The use something like $domain.tld as variable to merge the domain name you're referring to directly where variable is present in template. Easy and efficient.

PS: just noticed this thread https://github.com/PowerDNS-Admin/PowerDNS-Admin/discussions/1654

You can use variable [ZONE] to achieve that goal.
